.gitlab-ci.yml

```diff
@@ -21,16 +21,18 @@ variables:
     - apt-get install -y dpkg
     - apt-get install -y python-catkin-pkg python-rosdep python-wstool ros-${ROS_DISTRO}-catkin
     - apt-get install -y python3-pip python3-setuptools
+    - apt-get install -y python3-vcstool
     # Update setuptools from PyPI because the version Ubuntu ships with is too old
     - pip3 install -U setuptools
     - source /opt/ros/${ROS_DISTRO}/setup.bash
     - rosdep init
     - rosdep update
-    - cd ros/src
+    - mkdir -p $CI_PROJECT_DIR/../autoware.ai/src
+    - cd $CI_PROJECT_DIR/../autoware.ai
+    - vcs import src < $CI_PROJECT_DIR/autoware.repos
```

Comment:
This has the problem that if a pull request does not change the contents of autoware.repos, the master branches will be fetched.

Reply:
You could copy the original directory (`$CI_PROJECT_DIR`) over, so that the files that are tested are the ones from the pull request.